[2025-02-17T23:00:16.372+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-17T23:00:16.396+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: default_args_dag.task3 scheduled__2025-02-17T22:00:00+00:00 [queued]>
[2025-02-17T23:00:16.409+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: default_args_dag.task3 scheduled__2025-02-17T22:00:00+00:00 [queued]>
[2025-02-17T23:00:16.409+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-17T23:00:16.425+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): task3> on 2025-02-17 22:00:00+00:00
[2025-02-17T23:00:16.429+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=9181) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-17T23:00:16.431+0000] {standard_task_runner.py:72} INFO - Started process 9182 to run task
[2025-02-17T23:00:16.432+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'default_args_dag', 'task3', 'scheduled__2025-02-17T22:00:00+00:00', '--job-id', '986', '--raw', '--subdir', 'DAGS_FOLDER/12_default_args_dag.py', '--cfg-path', '/tmp/tmprh1e7e54']
[2025-02-17T23:00:16.433+0000] {standard_task_runner.py:105} INFO - Job 986: Subtask task3
[2025-02-17T23:00:16.478+0000] {task_command.py:467} INFO - Running <TaskInstance: default_args_dag.task3 scheduled__2025-02-17T22:00:00+00:00 [running]> on host ab74d4d90d32
[2025-02-17T23:00:16.560+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='test@test.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='default_args_dag' AIRFLOW_CTX_TASK_ID='task3' AIRFLOW_CTX_EXECUTION_DATE='2025-02-17T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-17T22:00:00+00:00'
[2025-02-17T23:00:16.561+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-17T23:00:16.575+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-17T23:00:16.576+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'sleep 5']
[2025-02-17T23:00:16.583+0000] {subprocess.py:99} INFO - Output:
[2025-02-17T23:00:21.587+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-17T23:00:21.617+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-17T23:00:21.618+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=default_args_dag, task_id=task3, run_id=scheduled__2025-02-17T22:00:00+00:00, execution_date=20250217T220000, start_date=20250217T230016, end_date=20250217T230021
[2025-02-17T23:00:21.683+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-17T23:00:21.697+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-17T23:00:21.699+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
